ndb/tests/unit/test_eventloop.py

```diff
@@ -22,16 +24,304 @@ def test___all__():
     tests.unit.utils.verify___all__(eventloop)
 
 
-def test_add_idle():
-    with pytest.raises(NotImplementedError):
-        eventloop.add_idle()
+def _Event(when=0, what='foo', args=(), kw={}):
+    return eventloop._Event(when, what, args, kw)
 
 
 class TestEventLoop:
     @staticmethod
-    def test_constructor():
+    def _make_one(**attrs):
+        """Use DummyClock"""
+        ev = eventloop.EventLoop(DummyClock())
```

Comment:
`ev` -> `event`? (2-letter variable names are not PEP 8 compliant and abbreviations can be tough when coming back to code)

Reply:
90ad15a